capstan.go

```diff
@@ -160,6 +160,16 @@ func main() {
 					return cli.NewExitError(err, EX_DATAERR)
 				}
 
+				bootOpts := cmd.SimpleBootOptions{
```

Comment:
Since I fail to recall the reason for prefixing with `Simple` I've removed it. What worries me is that I came up with this yesterday and forget it until today already...